src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -598,6 +604,23 @@ info.actuallyadditions.gui.pull=Pull
 info.actuallyadditions.gui.whitelist=Whitelist
 info.actuallyadditions.gui.blacklist=Blacklist
 info.actuallyadditions.gui.whitelistInfo=To let all items through, an empty blacklist can be used, to let no items through, an empty whitelist can be used. To configure certain items, place them or configured Item Filters in the slots.
+info.actuallyadditions.gui.respectMeta=Respecting Metadata
+info.actuallyadditions.gui.ignoreMeta=Ignoring Metadata
+info.actuallyadditions.gui.respectNBT=Respecting NBT
+info.actuallyadditions.gui.ignoreNBT=Ignoring NBT
+info.actuallyadditions.gui.ignoreMod=Mod Mode Off
+info.actuallyadditions.gui.respectMod=Mod Mode On
+info.actuallyadditions.gui.respectModInfo.1=If this is enabled, the filter will compare the mods items come from %s. This can be useful for storage systems with mod-based chests. Can also be combined with the other options, but that normally isn't very useful.
+info.actuallyadditions.gui.respectModInfo.2=instead of comparing the items themselves
+info.actuallyadditions.gui.ignoreOredict=Ignoring OreDictionary
+info.actuallyadditions.gui.respectOredictSoft=Soft Respecting OreDictionary
+info.actuallyadditions.gui.respectOredictHard=Hard Respecting OreDictionary
+info.actuallyadditions.gui.respectOredictInfo.one.1=The item being passed only has to contain %s of the OreDictionary tags of the item in the filter.
+info.actuallyadditions.gui.respectOredictInfo.one.2=only one
```

Comment:
This is a really bad way of doing it. Just use the vanilla formatting codes inside of the string above and remove the second part.
Same goes for the two ones below.